File: app/src/main/java/com/example/dronedeliveryservice/ActionsPage.java
package com.example.dronedeliveryservice;


import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;
import android.widget.TextView;import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View.OnClickListener;
import com.google.android.gms.maps.CameraUpdateFactory;
import com.google.android.gms.maps.GoogleMap;
import com.google.android.gms.maps.MapFragment;
import com.google.android.gms.maps.OnMapReadyCallback;
import com.google.android.gms.maps.SupportMapFragment;
import com.google.android.gms.maps.model.BitmapDescriptorFactory;
import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.Marker;
import com.google.android.gms.maps.model.MarkerOptions;


import androidx.appcompat.app.AppCompatActivity;
public class ActionsPage extends AppCompatActivity implements OnMapReadyCallback {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_actions_page);


        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
                .findFragmentById(R.id.map);
        mapFragment.getMapAsync(this);


    }

    @Override
    public void onMapReady(GoogleMap googleMap) {
        GoogleMap map_satellite = googleMap;
        map_satellite.setMapType(map_satellite.MAP_TYPE_SATELLITE);
        LatLng lexington = new LatLng(37.789426336756414, -79.43785182364378);
        map_satellite.animateCamera(CameraUpdateFactory.newLatLngZoom(lexington, 15f));
        googleMap.addMarker(new MarkerOptions()
                .position(lexington)
                .title("Marker at Mallory"))
                .setIcon(BitmapDescriptorFactory.fromResource(R.drawable.airplane_marker));
    }
}

File: app/src/main/java/com/example/dronedeliveryservice/ActionsPage.kt
package com.example.dronedeliveryservice

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_actions_page.*

class ActionsPage : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_actions_page)

        order_infoBtn.setOnClickListener{
            val intent1 = Intent(this, OrderInfo::class.java)
            startActivity(intent1)
        }
        bay1_btn.setOnClickListener{
            val intent2 = Intent(this, Bay1::class.java)
            startActivity(intent2)
        }
        dashboard_btn2.setOnClickListener{
            val intent3 = Intent(this, BayDashboard::class.java)
            startActivity(intent3)
        }

        dashboard_nav_button.setOnClickListener {
            val intent = Intent(this, BayDashboard::class.java)
            startActivity(intent)
        }

    }
}